.circleci/config.yml

```diff
@@ -276,15 +276,21 @@ commands:
     steps:
       - attach_workspace:
           at: .
+      - run:
+          name: Write cache suffix
+          command: |
+            echo '<<pipeline.parameters.cache-buster>>' > /tmp/react-get-deps.cachekey
+            cat webui/react/package-lock.json >> /tmp/react-get-deps.cachekey
+            echo $UID $HOME >> /tmp/react-get-deps.cachekey
```

Comment:
Are there $UIDs which share a $HOME? Feels like one or the other should be sufficiently unique. :)

Reply:
Oh right, this should be the current directory. I started off thinking of using home directory, then switched to current directory, but forgot to update some parts of my brain.